run.py

```diff
@@ -287,6 +287,13 @@ def run(args):
         if ubuntu_repo is None:
             ubuntu_repo = \
                 'http://download.eng.bos.redhat.com/rcm-guest/ceph-drops/3.3/latest-RHCEPH-3.3-Ubuntu/'
+    elif rhbuild.startswith('4'):
+        if base_url is None:
+            # default to latest RHCeph build 4.0
```

Comment:
@XDsubhash we may have to change the base URL. Since ceph 4.0 supports rhel-7.7. I suggest checking once if we have the new urls

Reply:
latest rhcs4 builds we got(sep13th) point to above base_url,will run against rhel 7.7 and check ,if it fails will request new repo links,as of now we are getting latest composes in the above url